src/main/java/player/FakeComputer.java

```diff
@@ -0,0 +1,22 @@
+package player;
+
+import move.Move;
+import move.MoveFactory;
+import move.Moves;
+
+import java.util.List;
+
+public class FakeComputer implements Player{
+
+    private MoveFactory moveFactory;
+
+    public FakeComputer() {
+        moveFactory = new MoveFactory();
+    }
+
+    public Move makeMove() {
+        List<String> moves = Moves.possibleMoves;
+        String randomMove = moves.get(0);
```

Comment:
Here you are always returning the first one for tests, so why not just hardcode the result to "return rock"?